README.md

```diff
@@ -1 +1,206 @@
+                        .to_log10(src=['time', 'derivative'], dst=['time', 'derivative'])
+                        .normalize(src=['time', 'derivative'],
+                                   dst_range=[None, 'derivative_q'])
+                        .get_samples(100, n_samples=1, sampler=np.random.random, src=['time', 'derivative'])
+                        .make_points(src=['time', 'derivative'], dst=['points'])
+                        .init_variable('predictions', init_on_each_run=list)
+                        .init_model('dynamic', TFModel, 'model',
+                                     config={'load' : {'path' : 'path_to_saved_model'},
+                                             'build': False})
+                        .init_variable('ind', init_on_each_run=list)
+                        .update_variable('ind', B('indices'), mode='e')
+                        .predict_model('model', fetches='predictions',
+                                                feed_dict={'points': B('points')},
+                                        save_to=B('predictions'), mode='w')
+                        .clip_values(src=['predictions'])
+                        .denormalize_component(src=['predictions'],
+                                               src_range=['derivative_q'])
+                        .update_variable('predictions', B('predictions'), mode='e')
+                    )
+    predicted_batch = (test_pipeline << dataset).next_batch(1)
+    predictions = predicted_batch.predictions
+```
+
+Or if you want to use a console. Run following command inside prod directory.
+
+> NOTE: This option works only with data that should be a NPY file with 2d numpy array with shape = (2, N).
+
+```bash
+foo@bar:~$ python predict.py -p ./path/to/data.npy -m /path/to/model (optional)
+```
+
+Process from preprocessing to prediction is described in this [notebook](./research/basic_usage.ipynb).
+
+## Model evaluation
+
+The detailed description of parameters and models estimation is written in this [notebook](./research/model_description.ipynb). The evaluation of best model is presented below. To estimate std we re-train model 20 times.
+
+MAPE < 30 % | STD
+------------ | -------------
+96.52 | 0.3086
+
+And the histogram of distribution of percentage error:
+
+<img src="./images/dist_of_ae.png" alt="drawing" width="520"/>
+
+## Installation
+
+### Installation as a project repository
+
+When cloning the repo from GitHub use flag ``--recursive`` to make sure that ``batchflow`` submodule is also cloned.
+
+    git clone --recursive https://github.com/analysiscenter/radial.git
+
+
+## Citing Radial
+
+Please cite Radial in your publications if it helps your research.
+
+
+    Khudorozhkov R., Broilovskiy A., Mylzenova D., Podvyaznikov D. Radial library for deep research
+    of finding exit point to the radial mode. 2019.
+
+```
+@misc{radial_2019_1156085,
+  author       = {Khudorozhkov R., Broilovskiy A., Mylzenova D., Podvyaznikov D.},
```

Comment:
Make sure that the format is OK. As far as I remember commas should be replaced to `and`

Reply:
I have replaced commas to ```and```.
What should we write after ```radial_2019```? We don't have any identifier yet.